src/documents/SubcomponentDeclaration.md

```diff
@@ -91,39 +96,48 @@ The properties allowed at top level in the subcomponent record are as follows:
+            <td><code>priority</code> [optional]</td>
+            <td><code>Number</code> or "first"/"last"</td>
+            <td>Specifies the order <a href="Priorities.md">priority</a> of the creation of the particular subcomponent. During component tree instantiation, the framework will sort the collection of subcomponents based on the priority specified. 
+                <div class="infusion-docs-note"><strong>Note:</strong> The use of this option should be discouraged as it leads to fragile configuration. If you find yourself using it, please report the instance to the development team to see if a better solution can be found.</div></td>
             <td><pre><code>subcomponent1: {
     type: "fluid.mySubcomponent",
     priority: "first"
 }</code></pre></td>
         </tr>
         <tr>
-            <td><code>container</code> (Required only for View components)</td>
-            <td>String</td>
-            <td>This property is a CSS-style selector identifying the container element to be used for this subcomponent. This property is required for any <a href="tutorial-gettingStartedWithInfusion/ViewComponents.md">View components</a>.</td>
+            <td><code>container</code> (Required only for view components)</td>
+            <td><code>String</code></td>
+            <td>This property is a CSS-style selector (or, more usually, an <a href="IoCReferences.md">IoC Reference</a> to an element produced by the <a href="DOMBinderAPI.md">DOM Binder</a> of some other component</a>) 
+            identifying the container element to be used for this subcomponent. This property is required for any <a href="tutorial-gettingStartedWithInfusion/ViewComponents.md">View components</a>.</td>
             <td></td>
         </tr>
     </tbody>
 </table>
 
 ## Injected Subcomponent Declaration ##
 
-Note that the entire subcomponent record may be replaced by a simple IoC reference to a component held elsewhere in the component tree. In this case the subcomponent is known as an _**injected component**_ - the already existing component reference is simply copied into the parent component's member field. Note that as a result of growing power in the IoC framework to perform most actions "in place" with respect to an existing component in its original location without having to inject it elsewhere. Should you find yourself using injected components, it is worth flagging the issue to the development team to see if a better solution could be found.
+Note that the entire subcomponent record may be replaced by a simple IoC reference to a component held elsewhere in the component tree. 
```

Comment:
I don't think this needs to be a note, maybe just remove "Note that" and start with "The entire subcomponent ... "